charts/federation-v2/README.md

```diff
@@ -83,13 +83,17 @@ chart and their default values.
 
 | Parameter | Description | Default |
 |-----------|-------------|---------|
-| controllermanager.replicaCount | Number of replica for federation v2 controller manager | 1 |
-| controllermanager.repository | Repo of the federation v2 image | quay.io/kubernetes-multicluster |
-| controllermanager.image | Name of the federation v2 image | federation-v2 |
-| controllermanager.tag | Tag of the federation v2 image | latest |
-| controllermanager.pullPolicy | Image pull policy | IfNotPresent |
-| clusterregistry.enabled | Specifies whether to enable the clusterregistry in federation v2 | true |
-| federatedtypes.enabled | Specifies whether to install the `FederatedTypeConfig` in federation v2 | true |
+| controllermanager.replicaCount | Number of replica for federation v2 controller manager. | 1 |
+| controllermanager.repository | Repo of the federation v2 image. | quay.io/kubernetes-multicluster |
+| controllermanager.image | Name of the federation v2 image. | federation-v2 |
+| controllermanager.tag | Tag of the federation v2 image. | latest |
+| controllermanager.pullPolicy | Image pull policy. | IfNotPresent |
+| controllermanager.featureGates | Feature gates are a set of `key=value` pairs that describe alpha or experimental features. An administrator can use the `--feature-gates` command line flag on each component to turn a feature on or off. | PushReconciler=true,SchedulerPreferences=true,CrossClusterServiceDiscovery=true,FederatedIngress=true |
+| controllermanager.fedNamespace | The namespace the federation control plane is deployed in. | federation-system |
+| controllermanager.clusterNamespace | The cluster registry namespace. | kube-multicluster-public |
+| controllermanager.clusterNamespace | Whether the federation namespace will be the only target for federation. | false |
+| clusterregistry.enabled | Specifies whether to enable the clusterregistry in federation v2. | true |
+| federatedtypes.enabled | Specifies whether to install the `FederatedTypeConfig` in federation v2. | true |
```

Comment:
s/federatedtypes/installfederatedtypes/ ?

I think it would be desirable to be explicit about what behavior is being enabled, which in this case is enabling the installation of federated types rather than enabling federated types themselves (which is accomplished via the installation of the FederatedTypeConfig CRD).

Reply:
I think the folder name should change, then.  As per my previous comment, 'enabling' federated types is an ambiguous action.

Reply:
@marun I was treating the `federatedtypes` as a sub chart and helm usually use noun for a chart name, but rename the folder to `installfederatedtypes` as sub chart name seems a bit strange?

Reply:
Nevermind, I don't think this will matter in light of #60, which implies that installation of federated types is likely to go away in favor of automation.